sbc_compassion/models/import_letters_history.py

```diff
@@ -426,7 +430,13 @@ def _find_template(self, img):
         """
         templates = self.env['sponsorship.correspondence.template'].search(
             [('pattern_image', '!=', False)])
-        return pr.find_template(img, templates)
+        return_value = pr.find_template(img, templates)
```

Comment:
`find_template` returns 2 values, you should capture them both or change method if the second return value is never used. Also use more specific variable name than `return_value`. Here it could be replaced by `templates`.